internal/service/federatedsettingsorgconfig/resource_federated_settings_connected_org_test.go

```diff
@@ -100,7 +110,7 @@ func configBasic(federationSettingsID, orgID, identityProviderID string) string
 		federation_settings_id     = "%[1]s"
 		org_id                     = "%[2]s"
 		domain_restriction_enabled = false
-		domain_allow_list          = ["reorganizeyourworld.com"]
+		domain_allow_list          = ["reorganizeyourworld.com", "cfn-test-domain.com"]
```

Comment:
nit: even though this is skipped, we could define an env variable so it is explicit that these values will depend on your local environment ([example](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/0140c87073a93a031f918de2b7d013af23aebb9f/internal/service/cluster/resource_cluster_test.go#L896)).

Reply:
Good point. Updated in [b4964e6](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2290/commits/b4964e64fcf3b7e5694bb425719387c29179e7b4)